VKSample/VKSample/NetworkLayer/NetworkService.swift

```diff
@@ -0,0 +1,89 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Network interface for VK API.
+protocol NetworkServiceProtocol {
+    func getFriends(userIDText: String)
+    func getPhotoUser(ownerID: Int)
+    func getGroupUser(userIDText: String)
+    func getSearchedGroup(text: String)
+}
+
+/// Network layer for VK API.
+final class NetworkService: NetworkServiceProvider {
+    // MARK: - Private Constants.
+
+    private enum Constants {
+        static let baseURL = "https://api.vk.com"
+        static let methodPathText = "/method/"
+        static let fieldsParamText = "fields"
+        static let accessTokenParamText = "access_token"
+        static let versionParamText = "v"
+        static let versionText = "5.131"
+        static let birthdayDateText = "bdate"
+        static let ownerIDText = "owner_id"
+        static let qText = "q"
+        static let friendsGetText = "friends.get"
+        static let photosGetAllText = "photos.getAll"
+        static let groupsGetText = "groups.get"
+        static let groupsSearchText = "groups.search"
+    }
+
+    // MARK: - Public methods.
+
+    func getFriends(userIDText: String) {
+        loadData(
+            method: Constants.friendsGetText,
+            parameterMap: [userIDText: String(Session.shared.userID)]
```

Comment:
```suggestion
            parameterMap: [userIDText: String(Session.shared.userID)]
```
параметры через URLComponets